packages/create-svelte/bin.js

```diff
@@ -136,6 +136,10 @@ async function main() {
 	} else if (options.types === 'checkjs') {
 		console.log(bold('✔ Type-checked JavaScript'));
 		console.log('  https://www.typescriptlang.org/tsconfig#checkJs');
+	} else if (options.template === 'skeletonlib') {
+		console.log(
+			'🛈 You chose to not add type checking, but TypeScript will still be installed in order to generate type definitions when building the library'
```

Comment:
what character is this supposed to be? it shows up empty in my terminal

Reply:
Huh, I saw it locally - it's supposed to be an info sign

Reply:
I replaced it with a warning sign, which looks better in combination with the check marks anyway - hopefully you see that one now

Reply:
i do, though the ⚠ character tends to look awful in a terminal context — on this MBP, where excellent font rendering is something you take for granted, it's a pixelated mess. I swapped it out for a yellow ▲ character instead